cmd/thanos/sidecar.go

```diff
@@ -345,7 +319,7 @@ func validatePrometheus(ctx context.Context, logger log.Logger, m *promMetadata)
 
 	// Check if block time is 2h.
 	if flags.TSDBMinTime != model.Duration(2*time.Hour) {
-		level.Warn(logger).Log("msg", "found that TSDB block time is not 2h. Only 2h block time is recommended.", "block-time", flags.TSDBMinTime)
+		level.Warn(logger).Log("msg", "Found that TSDB block time is not 2h. Only 2h block time is recommended.", "block-time", flags.TSDBMinTime)
```

Comment:
We stick in lower case log messages for all Prometheus ecosystem projects.

Reply:
done